core/src/cluster_nodes.rs

```diff
@@ -107,32 +108,42 @@ impl Node {
 
 impl<T> ClusterNodes<T> {
     pub(crate) fn submit_metrics(&self, name: &'static str, now: u64) {
+        let mut epoch_stakes = 0;
```

Comment:
maybe `stake_epoch` for consistency?

Reply:
I think `epoch_stakes` would be consistent with elsewhere in the code:
https://github.com/search?q=repo%3Asolana-labs%2Fsolana%20epoch_stakes&type=code
including where this value is set:
https://github.com/solana-labs/solana/blob/d4b30adff/core/src/cluster_nodes.rs#L425